perceval/utils/algorithms/circuit_optimizer.py

```diff
@@ -88,13 +88,15 @@ def max_eval_per_trial(self, value):
 
     def optimize(self,
                  target: ACircuit | Matrix,
-                 template: ACircuit
+                 template: ACircuit,
+                 empty_mode_list: list[int] = []
                  ) -> tuple[ACircuit, float]:
         """
         Optimize a template circuit unitary's fidelity with a target matrix or circuit.
 
         :param target: The target unitary circuit or matrix
         :param template: A circuit with variable parameters (supports only beam splitters and phase shifters)
+        :empty_mode_list: list of the modes without input photon, which are ignored during optmisation as this does not alter the computations
```

Comment:
Typo "optmisation" instead of "optimisation" (missing 'i')
Also, I would write "this does not alter the results"